README.md

```diff
@@ -1 +1,39 @@
-# demo
\ No newline at end of file
+# Fastify Official Demo
+
+![CI](https://github.com/fastify/demo/workflows/CI/badge.svg)
+[![js-standard-style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg?style=flat)](https://standardjs.com/)
+
+> :warning: **Please note:** This repository is still under active development.
+
+The aim of this repository is to provide a concrete example of a Fastify application using what are considered best practices by the Fastify community.
+
```

Comment:
Maybe explain which Fastify plugins are part of this demo?

Reply:
I don't think we need to maintain a list of plugins used in the doc. 

To be honest, I'm not really sure what the documentation should contain, although I'm aware that I should be working on it. IMO, I think it should be very minimalist, explaining the purpose of the demo and how to configure the application. The purpose of the demo is to show concrete code, so people should look at the code and comments.

Reply:
OK then not